src/navigation/checkout/SearchForm.js

```diff
@@ -46,6 +46,7 @@ const Autocomplete = (props) => {
 
   return <FlatList
     data={props.restaurants}
+    alwaysBounceVertical={false}
```

Comment:
Should this change only apply for iOS? 
If yes, you can use conditionals `alwaysBounceVertical={ Platform.OS === 'ios' ? false : true }`